### PR TITLE
[EOSF-878] Add packages for moving files to project

### PR DESCRIPTION
**Pull request by baylee-d**
## Purpose

Added ember packages necessary to get the move feature to work.

## Summary of Changes

Added:
- Ember power-select
- Ember collapsible-panels

## Ticket

https://openscience.atlassian.net/browse/EOSF-878

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] ~~changes described in `CHANGELOG.md`~~ *(don't have initial release yet)*

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -172,3 +172,5 @@ td .input-group {
 #toast-container > div {
     opacity: 1;
 }
+
+@import "ember-power-select";
```
Comment (jamescdavis) [inline app/styles/app.scss]: Should do this in addon/components/project-selector/style.scss in ember-osf

File: package.json [hunk 2]
```diff
@@ -51,12 +51,14 @@
     "ember-cli-template-lint": "^0.7.5",
     "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-collapsible-panel": "^2.1.1",
```
Comment (jamescdavis) [inline package.json]: should add this in ember-osf dependencies

File: package.json [hunk 3]
```diff
@@ -51,12 +51,14 @@
...
     "ember-data": "~2.16.0",
     "ember-export-application-global": "^1.0.5",
     "ember-font-awesome": "3.0.5",
     "ember-i18n": "^5.0.2",
     "ember-load-initializers": "^0.6.0",
     "ember-page-title": "3.2.2",
+    "ember-power-select": "^2.0.0-beta.0",
```
Comment (jamescdavis) [inline package.json]: should add this in ember-osf dependencies

Comment (jamescdavis) [reply package.json]: Also, shouldn't use 2.0.0-beta.0 if not absolutely necessary (use latest 1.x instead)

**Comment by baylee-d** [comment]
Closed because packages were moved to ember-osf.

**Pull request closed without merging**